src/main/java/com/icegreen/greenmail/user/UserManager.java

```diff
@@ -37,8 +37,8 @@ public GreenMailUser getUserByEmail(String email) {
         return ret;
     }
 
-    public GreenMailUser createUser(String name, String login, String password) throws UserException {
-        GreenMailUser user = new UserImpl(name, login, password, imapHostManager);
+    public GreenMailUser createUser(String name, String login, String password, boolean admin) throws UserException {

```

Comment:
add a method "createAdmin" would be cleaner than add a boolean there, it makes sense and avoid to provide unamed "true/false" values at many place